miden/src/tools/mod.rs

```diff
@@ -242,9 +241,9 @@ mod tests {
             "proc.foo.1 loc_store.0 end begin mem_storew.1 dropw push.17 push.1 movdn.2 exec.foo end";
         let stack_inputs = StackInputs::default();
         let advice_provider = MemAdviceProvider::default();
-        let program_info = super::analyze(source, stack_inputs, advice_provider)
+        let execution_details = super::analyze(source, stack_inputs, advice_provider)
             .expect("analyze_test: Unexpected Error");
-        let expected_program_info = ProgramInfo {
+        let expected_program_info = ExecutionDetails {
```

Comment:
nit: I'd probably rename the variable to `expected_details` or something like that.